ci.sh

```diff
@@ -183,6 +183,7 @@ cargo batch  \
     --- build --release --manifest-path tests/riscv32/Cargo.toml --target riscv32imac-unknown-none-elf \
     $BUILD_EXTRA
 
+rm out/tests/nrf52840-dk/wifi_esp_hosted_perf
```

Comment:
It would be nice to have a TODO comment, about the reason... :)

Reply:
done